src/mlpack/core/dists/regression_distribution.hpp

```diff
@@ -98,35 +99,36 @@ class RegressionDistribution
   void Train(const arma::mat& observations);
 
   /**
-   * Estimate parameters using provided observation weights
+   * Estimate parameters using provided observation weights.
    *
-   * @param weights probability that given observation is from distribution
+   * @param observations List of observations.
+   * @param weights probability that given observation is from distribution.
```

Comment:
I think the sentence starting after parameter argument should begin with capital letter word. Here and below.

Reply:
Oh, nice catch, and that is true.  When doxygen processes this it'll make a table with `weights` on the left side and the description on the right.  So it makes sense to have those be properly-capitalized sentences. :+1:

Reply:
Thanks for pointing it out, I'll edit it!!